data/lib/src/datasource_impl/shared_space_document_datasource_impl.dart

```diff
@@ -93,4 +103,30 @@ class SharedSpaceDocumentDataSourceImpl implements SharedSpaceDocumentDataSource
 
     return FileUploadState(mergedStream, UploadTaskId(taskId));
   }
+
+  @override
+  Future<List<WorkGroupNode>> getAllChildNodes(
+    SharedSpaceId sharedSpaceId, [WorkGroupNodeId parentNodeId]) {
+    return Future.sync(() async {
+      return (await _linShareHttpClient.getWorkGroupChildNodes(sharedSpaceId, parentNodeId))
+          .map<WorkGroupNode>((workgroupNode) {
+            if (workgroupNode is WorkGroupDocumentDto) return workgroupNode.toWorkGroupDocument();
+
+            if (workgroupNode is WorkGroupNodeFolderDto) return workgroupNode.toWorkGroupFolder();
+
+            developer.log('Cannot cast workgroup node = $workgroupNode');
+            return null;
+          }).toList();
+    }).catchError((error) {
+      _remoteExceptionThrower.throwRemoteException(error, handler: (DioError error) {
+        if (error.response.statusCode == 404) {
+          throw GetChildNodesNotFoundException();
+        } if (error.response.statusCode == 403) {
```

Comment:
are you sure about that?

Reply:
Can you express st?

Reply:
I don't found the Linshare Android have spec about 403, but I see some api are handling 403 error. I will check Linshare Core later.